inline_scan.sh

```diff
@@ -469,14 +472,31 @@ start_analysis() {
 		printf '\n%s\n' "***END SERVICE RESPONSE****" >&2
 	    fi
 	    exit 1
-	else
-	    if [ -f /tmp/sysdig/sysdig_output.log ]; then
-		cat /tmp/sysdig/sysdig_output.log
-	    fi
+	fi
+
+    FULLTAG="${SCAN_IMAGES[0]}"
+    check_status_with_digest
+}
+
+check_status_with_digest() {
+    RETRIES=100
+    # Then fetching the result of each scanned digest
+    for ((i=0;  i<${RETRIES}; i++)); do
+        status=$(curl -s -k  --header "Content-Type: application/json" -H "Authorization: Bearer ${SYSDIG_API_TOKEN}" "${SYSDIG_SCANNING_URL}/images/${IMAGE_DIGEST}/checkSummary?tag=$FULLTAG" | grep "status" | awk '{print $2}')
+        if [ ! -z  "$status" ]; then
+            break
         fi
+        echo -n "." && sleep 5
+    done
+ 
+    echo -n "Scan Report - "
```

Comment:
**nit**: I would print the new line here, so it's more convenient to copy the JSON:

```
Scan Report - {
  "imageDigest": "sha256:72c42ed48c3a2db31b7dafe17d275b634664a708d901ec9fd57b1529280f01fb",
  "at": "2019-10-10T08:24:15Z",
  "tag": "alpine",
  "status": "pass",
```